meson.build

```diff
@@ -0,0 +1,168 @@
+project('com.github.needleandthread.vocal', 'vala', 'c')
+
+meson.add_install_script('meson/post_install.py')
```

Comment:
https://mesonbuild.com/Gnome-module.html#gnomepost_install

Since meson 0.57 you can just do:

```meson
gnome = import('gnome')
gnome.post_install(glib_compile_schemas: true)

Reply:
good point, unfortunately, elementary (which I am building on) is stuck with 0.53.2

Reply:
Then I guess leave a comment for the future to remember to switch once the version is bumped. :D